mp4parse_capi/tests/test_avis_loop_count.rs

```diff
@@ -85,3 +102,12 @@ fn loop_forever() {
 fn no_edts() {
     check_loop_count("tests/no_edts.avif", -1);
 }
+
+#[test]
+fn check_timescales() {
+    check_timescale("tests/loop_1.avif", 2);
+    check_timescale("tests/loop_2.avif", 2);
+    check_timescale("tests/loop_ceiled_4.avif", 2);
+    check_timescale("tests/loop_forever.avif", 2);
```

Comment:
Weird but correct timescale. I haven't been able to figure out yet what tool is able to create an `avis` file that has a top-level timescale and a track timescale, for testing, `ffmpeg` doesn't.